permission_handler/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 8.1.4
+
+* Fix bug where `locationAlways` returned sometimes `PermissionStatus.Denied` instead of `PermissionStatus.Granted`.
```

Comment:
nit: small grammar mistake. It would be more correct to say:
```suggestion
* Fix bug where requesting `locationAlways` permission sometimes returns `PermissionStatus.Denied` instead of `PermissionStatus.Granted`.
```